smartsim/_core/launcher/local/local.py

```diff
@@ -104,16 +115,23 @@ def run(self, step: Step) -> str:
             self.task_manager.start()
 
         out, err = step.get_output_files()
-        output = open(out, "w+")
-        error = open(err, "w+")
-        cmd = step.get_launch_cmd()
+
+        # pylint: disable-next=consider-using-with
+        output = open(out, "w+", encoding="utf-8")
+        # pylint: disable-next=consider-using-with
+        error = open(err, "w+", encoding="utf-8")
 
         # LocalStep.run_command omits env, include it here
         passed_env = step.env if isinstance(step, LocalStep) else None
 
+        cmd = step.get_launch_cmd()
+        if os.environ.get("SSFLAG_TELEMETRY", False):
```

Comment:
@MattToast suggestions on env var?

Reply:
I think the name is great! It leaves it open for other additional `SSFLAG_*` vars in future which I like!

One small note: if  a user runs `export SSFLAG_TELEMETRY=0` this condition will evaluate true bc `"0"` is a non-empty string. Should we update this so that it first casts the env to an int to handle that case?

Reply:
per later convo, renamed.